### ISSUE TITLE
Improvement for k8s.io/docs/concepts/overview/working-with-objects/common-labels/

### ISSUE THREAD

**Issue by valetarton** [kubernetes/website#25857]
```
With the MySQL StatefulSet and Service you'll notice information about both MySQL and Wordpress, the broader application, are included.
```

Last instance of "WordPress" is spelled without a capital "P" as it is spelled on the rest of the page within the context of wordpress.com

**Comment by Aut0R3V** [comment]
@valetarton I had a look at it, I found only one line which has been included in the issue where WordPress was mentioned as Wordpress. I've sent in a pull request for this.

**Comment by sftim** [comment]
/good-first-issue
addressed in #25868

/close

**Comment by sftim** [comment]
/triage accepted
too
